torch_np/random.py

```diff
@@ -29,21 +30,44 @@
     "randint",
     "shuffle",
     "uniform",
+    "RANDOM_STREAM",
 ]
 
 
+RANDOM_STREAM = "pytorch"
```

Comment:
Just make it a boolean variable. Something like `USE_NUMPY_RANDOM = False`

Reply:
The string value here parallels `set_default_dtype`: https://github.com/Quansight-Labs/numpy_pytorch_interop/blob/main/torch_np/_dtypes.py#L346

Reply:
I still think it's better to have a boolean flag for this, as we just have two values. In the other case we also have other possible values, so the enum is alright. Although I'm not the biggest fan of the enum sugar in the dtypes either, but that's a story for another day ajjaja

Reply:
OK, will change. My experience is that almost all boolean flags (modulo a set of measure zero, that is) have to be changed to more than two state eventually. But here it's OK for now indeed, so let me change it and move on :-).